### PR TITLE
Create page summarizing service types

### PR DESCRIPTION
**Pull request by justindeguzman**
Support has requested a page to point to prospective customers describing the different service types in more detail

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit 774d64f5686d191790f7c60ad3e4b40351b1fe27 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/1519/774d64f5686d191790f7c60ad3e4b40351b1fe27/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>✅ success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: docs/en/cloud/manage/service-types.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+|**Compute**|Burstable CPU|Dedicated CPU|Custom compute options|
+|**Backups**|Every 24h, retained for 1 day|Every 24h, retained for 2 days|Custom backup retention|
+|**Upgrades**|Automatic|Automatic|Schedulable|
+|**SLA and support**|24-hour response time|1-hour response time|Custom SLAs, assigned lead support engineer|
+
+## Development
+
+`Development` services are designed for smaller workloads and starter projects. They are the lowest-cost option in ClickHouse Cloud. Though at a lower price than our other service types, `Development` services are still designed for high reliability and are replicated across two availability zones.
+
+`Development` services do not support autoscaling and are “auto on” by default. `Development` services are best for internal projects and prototypes, and for developers just trying out ClickHouse.
```
Comment (tbragin) [inline docs/en/cloud/manage/service-types.md:27]: > and are “auto on” by default.

@justindeguzman what does this mean? idling is on by default both in Development and Prod

**Comment by tbragin** [comment]
I added one minor comment - other than that LGTM! Thank you!

**Review by tbragin — CHANGES_REQUESTED** [review]
I added one minor comment but other than that LGTM

File: docs/en/cloud/manage/service-types.md (hunk 1 above)
Comment (justindeguzman) [reply docs/en/cloud/manage/service-types.md:27]: Sounds good, I'll remove `and are “auto on” by default`

**Pull request merged**